tests/unit/components/about.spec.js

```diff
@@ -0,0 +1,62 @@
+import Vue from 'vue'
+import { mount, createLocalVue } from '@vue/test-utils'
+import Vuetify from 'vuetify'
+import VueX from 'vuex'
+import { cloneDeep } from 'lodash'
+import About from '@/views/About.vue'
+import { PEER_CONNECTED } from '@/store/mutation-types'
+import edgeDevice from '@/store/edge-device.js'
+import pnpStoreModule from '@/store/pnp'
+
+describe('About Page', () => {
```

Comment:
@vickywane About page only shows the PWA UI app version, so tests should reflect that. No point in testing for edge device version because its not shown on the About page.